tests/test_rpm_verifier.py

```diff
@@ -125,13 +128,42 @@ def test_get_unsigned_rpms(test_input: list[str], expected: list[str]) -> None:
     assert result == expected
 
 
+@pytest.mark.parametrize(
+    "test_input,expected",
+    [
+        pytest.param(
+            [
+                "libssh-config-0.9.6-10.el8_8 RSA/SHA256, Tue 6 May , Key ID 1234567890",
+                "python39-twisted-23.10.0-1.el8ap (none)",
+                "libmodulemd-2.13.0-1.el8 RSA/SHA256, Wed 18 Aug , Key ID 1234567890",
+                "gpg-pubkey-d4082792-5b32db75 (none)",
+            ],
```

Comment:
Would be good to add another line with a different key ID so we can see it can return more than a single value :)